packages/ksf-subscription/src/KSF/Subscription/Component.purs

```diff
@@ -56,8 +56,13 @@ render { props } =
   View.subscription
     { product: props.subscription.package.name
     , status: translateStatus props.subscription.state
-    , nextBillingDate: trim $ fromMaybe "" $ formatDate =<< addOneDay props.subscription.dates.end
+    , nextBillingDate: nextBillingDate
     }
+  where
+    nextBillingDate
+      | Persona.isSubscriptionCanceled props.subscription = Nothing
+      | otherwise =
+          Just $ trim $ fromMaybe "" $ formatDate =<< addOneDay props.subscription.dates.end
```

Comment:
Is there a need to do `fromMaybe ""` here? It could be just `map trim $ formatDate =<< addOneDay props.subscription.dates.end`.

Reply:
Indeed